kubernetes-client/src/com/goyeau/kubernetes/client/api/PodsApi.scala

```diff
@@ -121,9 +113,13 @@ private[client] class NamespacedPodsApi[F[_]](
       .+??("container" -> container)
       .++?("command" -> commands)
 
-    execHeaders.map { execHeaders =>
-      WSRequest(uri, execHeaders, Method.POST)
-    }
+    WSRequest(uri, method = Method.POST)
+      .withOptionalAuthorization(authorization)
+      .map { r =>
+        r.copy(
```

Comment:
nit `_` instead of `r => r`?

Reply:
I was wondering why I didn't do that initially :) 

But `r` is referenced twice there (`r.headers.put`)